common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -70,7 +71,7 @@ public BrokerRequest brokerRequestFromSilentOperationParameters(@NonNull final A
                 .localAccountId(parameters.getAccount().getLocalAccountId())
                 .username(parameters.getAccount().getUsername())
                 .claims(parameters.getClaimsRequestJson())
-                .forceRefresh(parameters.getForceRefresh())
+                .forceRefresh(parameters.getForceRefresh() || !TextUtils.isEmpty(parameters.getClaimsRequestJson()))
```

Comment:
These changes are related or fixing another issue?

Reply:
yes , have a vsts issue, will link the PR there